src/app.js

```diff
@@ -0,0 +1,247 @@
+/* import { API_KEY as key } from './variables.js'; //Import and Rename
+console.log(key); */
+let spinner = document.querySelector(".spinner"); //Spinner to show when fetching api data
```

Comment:
If we are using Spinner as a class and other helpers as methods, I wouuld say we need to remove it from here

Reply:
Removed redundant code